src/dynamics/rigid_body.rs

```diff
@@ -297,6 +298,11 @@ impl RawRigidBodySet {
         self.map(handle, |rb| RawVector(*rb.angvel()))
     }
 
+    /// The velocity of the given world-space point on this rigid-body.
+    pub fn rbVelocityAtPoint(&self, handle: FlatHandle, point: &RawVector) -> RawVector {
+        self.map(handle, |rb| rb.velocity_at_point(&Point::from(point.0)).into())
```

Comment:
Not sure if `&Point::from(point.0)` is the right approach? Tried `point.0.into()` like in other places but the compiler wasn't happy with that